CONTRIBUTING.md

```diff
@@ -70,6 +70,27 @@ manager, you will need to find and install the following tools:
 - JDK 8
 - `sbt`
 - `z3`
+- `gmp`
+- `mpfr`
+
+After installing the tools listed above, you need to download Apron source code,
```

Comment:
Let's list here just the Apron dependency (link to the project or something).

Then, move this to a section on Apron installation.